tests/testthat/test-metadata.R

```diff
@@ -638,8 +638,8 @@ test_that("xportr_type: Variable types are coerced as expected and can raise mes
 # tests for `xportr_metadata()` basic functionality
 # start
 test_that("xportr_metadata: Check metadata interaction with other functions", {
-  skip_if_not_installed("admiral")
-  adsl <- admiral::admiral_adsl
+  # making sure data is being loaded from xportr namespace (name may conflict)
+  adsl <- xportr::adsl
```

Comment:
this gives me pause...should our adsl object get callsed `xportr_adsl` or `adsl_xportr`

Reply:
I tend to agree as name clashing with `adsl` is prone to happen.

Reply:
Thumbs up if you want to change :grin: 

This should only affect Vignettes and "possibly" examples.

Reply:
yes lets change it please